app/samples/phase_detection/src/flight.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+#include <array>
+#include <cstdint>
+#include <f_core/flight/CPhaseController.h>
+
+/**
+ * Flight Events for a fictional flight
+ */
+enum Events : uint8_t { PadReady, Boost, Coast, Noseover, MainChute, GroundHit, CamerasOff, NumEvents };
+constexpr std::array<const char *, Events::NumEvents> eventNames = {
+    "PadReady", "Boost", "Coast", "Noseover", "MainChute", "GroundHit", "CamerasOff",
+};
+
+/**
+ * Sources of flight events
+ */
+enum Sources : uint8_t {
+    IMU1,
+    Barom1,
+    NoseoverLockout,
+    Boost2CoastTimer,
```

Comment:
Was considering making Timers their own type of source and making Source only sensors but having Timers have their own source value allows you to say if Boost2CoastTimer AND Barom1 detects coast, then do it.

I think it makes sense to keep them all as one type for that reason